pkg/arvo/lib/metadata-json.hoon

```diff
@@ -1,24 +1,26 @@
 /-  *metadata-store
+/+  resource
+^?
```

Comment:
File name should be changed from `metadata-json` to `metadata-store` to match the other agent libraries. We should also update this file while we're at it to follow the `enjs` and `dejs` pattern seen elsewhere.